reference/tax_provider.yml

```diff
@@ -1332,6 +1285,42 @@ components:
         - rate
         - amount
       x-internal: false
+    item_type:
+      type: string
+      description: |-
+        The type of item for the line item in the document.
+
+        Tax estimate requests for order-level refunds use the type `refund`.
+      enum:
+        - item
+        - refund
```

Comment:
@theromulans  When a request is made for an order-level refund, is `refund` the `type` for all the line items? Is `refund` an allowed value for **each** type of line item (e.g. wrapping line items, handling line items, etc)?

Reply:
An order-level refund is an additional item outside the bounds of the order - it doesn't affect any other items. It is included as a way of simply informing the tax provider, if they are interested in the information - generally for accounting purposes.

So, yes `refund` is only for order-level refunds, a completely separate item type. Handling, shipping, wrapping cannot be a `refund`. So, less an `enum` there and more an exact value of `handling`, `shipping` or `wrapping`.

Reply:
Thanks @theromulans ! I removed `refund` from handling, shipping, and wrapping. Let me know if there's anything else